### PR TITLE
increase controller workers for available and cleanup

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: pkg/spoke/spokeagent.go [hunk 1]
```diff
@@ -26,6 +26,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 )
 
+const (
+	// TODO expose a flag to set the worker for each controller
+	appliedManifestWorkFinalizeControllerWorkers = 10
```
Comment (qiujian16) [inline pkg/spoke/spokeagent.go:36]: can we add a comment on why it needs to be 10?

Comment (skeeey) [reply pkg/spoke/spokeagent.go:36]: added, PTAL

**Pull request merged**